static/bidder-info/adelement.yaml

```diff
@@ -0,0 +1,16 @@
+endpoint: "https://amp.adelement.com/openrtb2/auction?supply_id={{.SupplyId}}"
+endpointCompression: gzip
+maintainer:
+  email: "tech@adelement.com"
```

Comment:
sent email for verification.

Reply:
i have just confirm with "received" reply

Reply:
email confirmed. thanks!